docs/source/contributing/documentation.rst

```diff
@@ -318,6 +318,23 @@ elements that are either a ``str`` or ``None``;
 ``(str, int)``; ``Tuple[:class:`int`, ...]`` for a tuple of variable
 length with only integers.
 
+.. note::
+   As a helper for tool for typesets, you can use ``typestring-parser`` which can 
+   be accessed by first downloading it via ``pip`` - ``pip install typestring-parser`` and
+   then using ``from typestring_parser import parse``.
+
+.. code-block:: bash
```

Comment:
how is the below code `bash`? It's python.

Reply:
We use it under bash or cmd so it's not python right?

Reply:
Again, see how this is done in other places. It's obviously not `bash`.

Reply:
I made another change and checked locally which passed but over here the pre commit is failing